### PR TITLE
[Robot][GE] Delete/renamed fields error verification tests

### PR DESCRIPTION
**Pull request by skristem**
Tests Added:
robot/Cumulus/tests/browser/gift_entry/error_validation_delete_fields.robot
Deletes a field that's mapped, verifies warnings, and errors are thrown on AM, Object group page, template builder. Verify that saving the gift throws an error. Creates the field and verifies no errors or warnings are thrown on AM, object group, and template builder. Verifies that a gift can be created successfully.

robot/Cumulus/tests/browser/gift_entry/error_validation_rename_fields.robot
Creates a template with a custom field on template form, renames the field, verifies warnings and error are thrown on AM, object group page, template builder. Verify field is not present on the batch gift form. Save the gift and verify the gift status is dry run error. Rename the field to correct it and verify no errors or warnings are thrown on AM, object group, and template builder. Creates a batch and verifies field is back on form and on saving gift its status is dry run validated.

To execute Tests:

cci task run robot -o  suites robot/Cumulus/tests/browser/gift_entry/<test_name> --org <org_name>

# Critical Changes

# Changes

# Issues Closed

# Community Ideas Delivered

# New Metadata

# Deleted Metadata

### CODE REVIEW

**Review by deepmalamehta — APPROVED** [review]

**Review by bdvorachek — APPROVED** [review]
Both new tests passing in my scratch org, looks good!

**Review by smparedes — APPROVED** [review]
@skristem good work!

File: robot/Cumulus/resources/ObjectMangerPageObject.py [hunk 1]
```diff
@@ -187,4 +189,56 @@ def add_picklist_values(self, **kwargs):
...
+			search_result = npsp_lex_locators['object_manager']['search_result'].format(field_name)
+			self.selenium.click_element(search_result)
+			self.npsp.select_frame_and_click_element("Account Custom Field","object_manager.button","Edit")
+			self.selenium.unselect_frame()
+			self.npsp.wait_for_locator('frame_new', 'Edit Account Custom Field', 'Edit Account Custom Field')
+			self.npsp.choose_frame('Edit Account Custom Field')
+			self.selenium.wait_until_page_contains_element(save_button)
+			locator = self.salesforce._get_input_field_locator('Field Label')
+			element = self.selenium.get_webelement(locator)
+			self.selenium.driver.execute_script("arguments[0].setAttribute(arguments[1], arguments[2]);", element, "onChange", "")
```
Comment (rjanjanam) [inline robot/Cumulus/resources/ObjectMangerPageObject.py:220]: What does this jS execution block do ?

**Review by rjanjanam — APPROVED** [review]

**Review by schlossmj — APPROVED** [review]

File: robot/Cumulus/resources/ObjectMangerPageObject.py (hunk 1 above)
Comment (skristem) [reply robot/Cumulus/resources/ObjectMangerPageObject.py:220]: it is going to disable the alert that popsup when u change the value in field

**Pull request merged**